src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -70,6 +70,7 @@
       <!-- Create a list of all content that *is* in the wwwroot folder (to preserve ItemGroup metadata) -->
       <_TemporaryHiddenContent Include="@(Content)" Exclude="@(_NonWWWRootContent)" />
       <!-- Add Scoped CSS files in the app to the list of hidden items -->
+      <_TemporaryHiddenContent Include="Components\**\*.razor.css" />
```

Comment:
Does mean you really can only have Blazor components with component specific styles (.razor.css) in these folders?

Reply:
I _believe_ so, at least, that's what it originally was for. But in the intervening years no one has ever complained about it even once...

Having said that, I'm not entirely sure it's necessary, but I feel it's quite safe to add (that is, it shouldn't cause any harm, and at most should help some scenarios).

Reply:
Why does the same thing appear on lines 73 and 95? Is that an accident?

Reply:
The top one is a target for not-hot-restart, the bottom one is hot-restart specifically. At one point we had only the not-hot-restart target and the fix was to duplicate everything into another target.